.travis.yml

```diff
@@ -2,8 +2,23 @@ language: node_js
 node_js:
   - "8.10"
   - "10"
+before_install:
+  - npm install -g yarn
```

Comment:
as far as I can tell, the presence of `yarn.lock` means that yarn will already be available ([docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#yarn-is-supported))

Reply:
The one that Travis preinstalls is old and doesn't support `yarn workspace` command. `npm install -g yarn` is to upgrade the yarn version.